### DISCUSSION TITLE
glitch

### DISCUSSION THREAD

**Discussion by jbxskm** [BigfootsGS/BigfootsGS.github.io#44] [Q&A] [answered]
can you make this in glitch then send the link so i can remix it?

**Comment by jbxskm** [comment]
this website is epic by the way

**Reply by Bigfoot9999** [reply]
> this website is epic by the way

Thanks bro :)

**Comment by Bigfoot9999** [comment, marked as answer]
> can you make this in glitch then send the link so i can remix it?

Go to this link: https://glitch.com/edit/#!/import/github/BigfootsGS/BigfootsGS.github.io

**Comment by jbxskm** [comment]
[Screen recording 2023-01-20 9.25.24 AM.webm](https://user-images.githubusercontent.com/111071747/213741497-e3aa3428-c3cd-4a9c-96a3-8b0400aad6be.webm)
Last stand union city is weird
theres a yellow screen
i submited a google form heres the video

**Comment by jbxskm** [comment]
this game is looping
i cant play it
[Screen recording 2023-01-20 11.10.35 AM.webm](https://user-images.githubusercontent.com/111071747/213761527-1cffbcf8-236e-41be-8d41-d4dcdc0cdaea.webm)

**Reply by Bigfoot9999** [reply]
At the moment we don't have enough devs to fix games. If you can find the file in a flash archive, open a pull request and I'll check it out.
Sorry for the inconvenience.
